src/test/librbd/test_internal.cc

```diff
@@ -763,28 +763,28 @@ TEST_F(TestInternal, SnapshotCopyup)
       bufferlist expected_bl;
```

Comment:
It would be good to adjust `rbd_sparse_read_threshold_bytes` setting a few lines above as well.  It doesn't affect the execution of the test but would prevent people from wondering "why 256 and not 4096" in the future.